### ISSUE TITLE
Can't compile evision 0.1.8 on MacOS (Intel)

### ISSUE THREAD

**Issue by kipcole9** [cocoa-xu/evision#96]
@cocoa-xu this is looking really good but I think there may be an issue with `0.1.8`.  I have no issues with `0.1.7` on either MacOS Intel or MacOS ARM but with `0.1.8` I see two issues:

1. Issues with TLS peer verification (I have some [code that might help](https://github.com/elixir-cldr/cldr_utils/blob/master/lib/cldr/http/http.ex) if you're using `:httpc`)
2. What looks like an asset missing from the package?

```elixir
kip@Kips-iMac-Pro image % iex -S mix                                                           
Erlang/OTP 25 [erts-13.1] [source] [64-bit] [smp:16:16] [ds:16:16:10] [async-threads:1] [jit:ns]


11:33:28.673 [info] EVISION_PREFER_PRECOMPILED: true; try to download and use the precompiled library.
 
11:33:28.676 [info] Requested version `0.1.8` has precompiled binaries.

11:33:28.676 [info] Current target `x86_64-apple-darwin` has precompiled binaries.
 
11:33:28.705 [warning] Description: 'Authenticity is not established by certificate path validation'
     Reason: 'Option {verify, verify_peer} and cacertfile/cacerts is missing'

 
11:33:29.115 [warning] Description: 'Authenticity is not established by certificate path validation'
     Reason: 'Option {verify, verify_peer} and cacertfile/cacerts is missing'

 
11:33:33.109 [info] Precompiled binary tarball downloaded and saved to /Users/kip/Library/Caches/evision-x86_64-apple-darwin-0.1.8.tar.gz, sha256=2af7710079074f607342717a0a7a6e6d6d90aca2b7038420636602954f6afa65
==> evision
could not compile dependency :evision, "mix compile" failed. Errors may have been logged above. You can recompile this dependency with "mix deps.compile evision", update it with "mix deps.update evision" or clean it with "mix deps.clean evision"
** (MatchError) no match of right hand side value: {:error, "the precompiled NIF file does not exist in the checksum file. Please consider run: `EVISION_FETCH_PRECOMPILED=true mix evision.fetch --all` to generate the checksum file."}
    /Users/kip/Development/image/deps/evision/mix.exs:298: Mix.Tasks.Compile.EvisionPrecompiled.prepare/3
    (mix 1.14.0) lib/mix/task.ex:421: anonymous fn/3 in Mix.Task.run_task/4
    (mix 1.14.0) lib/mix/tasks/compile.all.ex:92: Mix.Tasks.Compile.All.run_compiler/2
    (mix 1.14.0) lib/mix/tasks/compile.all.ex:72: Mix.Tasks.Compile.All.compile/4
    (mix 1.14.0) lib/mix/tasks/compile.all.ex:59: Mix.Tasks.Compile.All.with_logger_app/2
    (mix 1.14.0) lib/mix/tasks/compile.all.ex:33: Mix.Tasks.Compile.All.run/1
    (mix 1.14.0) lib/mix/task.ex:421: anonymous fn/3 in Mix.Task.run_task/4
    (mix 1.14.0) lib/mix/tasks/compile.ex:134: Mix.Tasks.Compile.run/1
```

**Comment by cocoa-xu** [comment]
Hi @kipcole9, `:evision` is now released on hex.pm, and the checksum file, `checksum-evision.exs`, is only tracked in hex package. Please using the hex.pm one. :)

As for the TLS one, I guess I can steal some code from your `elixir-cldr` library XD. I'll add them in the next version!

**Comment by cocoa-xu** [comment]
**TL; DR:** basically, it's now possible and recommended to use `:evision` as a normal hex package.

When using `:evision` from git, version "0.1.1" to "0.1.7" are available.

Starting from `0.1.8` (included), using `:evision` from git is no longer supported (unless set `EVISION_PREFER_PRECOMPILED` to `false`) because we started to use checksum file to verify the integrity of the downloaded tarball file, and the checksum file is only tracked in the hex.pm package.

```elixir
def deps do
  [
    {:evision, "~> 0.1.7", github: "cocoa-xu/evision", tag: "v0.1.7"}
  ]
end
```

It is recommended to use `:evision` from hex.pm. Currently version "0.1.7" to "0.1.8" are available on hex.pm,
```elixir
def deps do
  [
    {:evision, "~> 0.1.8"}
  ]
end
```

**Comment by kipcole9** [comment]
That fantastic, I didn't notice that from the commits (my mistake). All good now, congratulations on a first hex release!

**Comment by cocoa-xu** [comment]
> That fantastic, I didn't notice that from the commits (my mistake). All good now, congratulations on a first hex release!

Thanks and please don't worry about that, I was expecting issues like this. And I haven't announced that on my Twitter or elsewhere because I kinda see that it would be somewhat bumpy in the first few hex releases for this project.
